features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/media/local/LocalMediaView.kt

```diff
@@ -186,3 +212,50 @@ fun MediaPDFView(
     }
     PdfViewer(pdfViewerState = pdfViewerState, modifier = modifier)
 }
+
+@Composable
+fun MediaFileView(
+    uri: Uri?,
+    info: MediaInfo?,
+    onReady: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    LaunchedEffect(Unit) {
+        if(uri != null) {
+            onReady()
+        }
```

Comment:
Probably out of scope for this PR, but this kind of callback feels a bit weird to me.

I think the more typical way would be to pass in a mutable state object, like how a `LazyColumn` takes a `LazyListState` that is populated with its state?

Reply:
I agree, I can change that.